pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -16,16 +16,25 @@ import (
 )
 
 const (
-	keyPattern   = "%s_%s"
-	invKeyPrefix = "inv_"
+	keyPattern        = "%s_%s"
+	invKeyPrefix      = "inv"
+	connIDKeyPrefix   = "conn"
+	protocolKeyPrefix = "pro"
 )
 
 // ConnectionRecord contain info about did exchange connection
 type ConnectionRecord struct {
 	// State of the connection invitation
-	State string
-
-	ConnectionID string
+	State           string
+	ConnectionID    string
+	ThreadID        string
+	ID              string
```

Comment:
What is this ID?

Reply:
@troyronda . I followed this design discussion we had which is recorded in this issue. Here ID is request ID. I have renamed it to RequestID
https://github.com/hyperledger/aries-framework-go/issues/226#issuecomment-536062767